moveit_ros/planning/kinematics_plugin_loader/include/moveit/kinematics_plugin_loader/kinematics_plugin_loader.h

```diff
@@ -57,8 +58,17 @@ class KinematicsPluginLoader
   KinematicsPluginLoader(const rclcpp::Node::SharedPtr& node,
                          const std::string& robot_description = "robot_description",
                          double default_search_resolution = 0.0)
-    : node_(node), robot_description_(robot_description), default_search_resolution_(default_search_resolution)
+    : robot_description_(robot_description), default_search_resolution_(default_search_resolution)
   {
+    node_interface_ = std::make_shared<node_interface::NodeInterface>(node_interface::NodeInterface(node));
+  }
```

Comment:
Put the node interface in the initalizer list with the rest of members.

Reply:
I made the fix.